README.md

```diff
@@ -23,149 +9,148 @@ provided for getting started quickly.
 [![codecov](https://codecov.io/gh/pymor/pymor/branch/main/graph/badge.svg)](https://codecov.io/gh/pymor/pymor)
 [![pre-commit.ci status](https://results.pre-commit.ci/badge/github/pymor/pymor/main.svg)](https://results.pre-commit.ci/latest/github/pymor/pymor/main)
 
+# pyMOR - Model Order Reduction with Python
 
-License
--------
-
-Copyright pyMOR developers and contributors. All rights reserved.
-
-Redistribution and use in source and binary forms, with or without modification, are permitted provided that the
-following conditions are met:
+pyMOR is a software library for building
+[model order reduction](https://morwiki.mpi-magdeburg.mpg.de)
+applications with the Python programming language.
+All algorithms in pyMOR are formulated in terms of abstract interfaces,
+allowing generic implementations to work with different backends,
+from NumPy/SciPy to external partial differential equation solver packages.
 
-* Redistributions of source code must retain the above copyright notice, this list of conditions and the following
-  disclaimer.
-* Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following
-  disclaimer in the documentation and/or other materials provided with the distribution.
+## Features
 
-THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
-INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
-DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
-SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
-SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
-WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
-THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+* Reduced basis methods for parametric linear and non-linear problems.
+* System-theoretic methods for linear time-invariant systems.
+* Neural network-based methods for parametric problems.
+* Proper orthogonal decomposition.
+* Dynamic mode decomposition.
+* Rational interpolation of data (Loewner, AAA).
+* Numerical linear algebra (Gram-Schmidt, time-stepping, ...).
+* Pure Python implementations of finite element and finite volume
+  discretizations using the NumPy/SciPy scientific computing stack.
```

Comment:
Do we want to be more specific here? For instance, one could mention which system-theoretic methods are actually implemented or that things like greedy, empirical interpolation and error estimation are available... However, that might also be too much, I am not sure.

Reply:
Maybe let's discuss this elsewhere and first merge this pr, @pmli, @HenKlei?

Reply:
Fine with me.